packages/vm/package.json

```diff
@@ -16,12 +16,12 @@
     "coverage:test": "tape './tests/api/**/*.js' ./tests/tester.js --state --dist",
     "docs:build": "typedoc --options typedoc.js",
     "test:state": "ts-node ./tests/tester --state",
-    "test:state:allForks": "echo 'Homestead TangerineWhistle SpuriousDragon Byzantium Constantinople Petersburg Istanbul MuirGlacier' | xargs -n1 | xargs -I v1 npm run test:state -- --fork=v1",
+    "test:state:allForks": "echo 'Chainstart Homestead dao TangerineWhistle SpuriousDragon Byzantium Constantinople Petersburg Istanbul MuirGlacier' | xargs -n1 | xargs -I v1 npm run test:state -- --fork=v1",
```

Comment:
The convention here for the HF names is normally to apply camel case, so at least *start* with a capital letter (`Dao`) (so but for dao it is a bit unclear to admit, could also be `DAO`). We shouldn't deviate on such things too much since these kind of things always have the potential to lead to some not-wanted side effects at some point.

Won't make this a review blocker though.